src/de/jost_net/JVerein/gui/view/MailVorlageDetailView.java

```diff
@@ -54,4 +66,49 @@ public void handleAction(Object context)
     }, null, true, "document-save.png");
     buttons.paint(this.getParent());
   }
+
+  @Override
+  public void handleEvent(Event event)
+  {
+     if (event.keyCode == org.eclipse.swt.SWT.F6)
+    {
+      new OpenInsertVariableDialogAction().handleAction(null);
+    }
+  }
+
+  private static final class OpenInsertVariableDialogAction implements Action
+  {
+
+    @Override
+    public void handleAction(Object context)
+    {
+      try
+      {
+        LesefeldAuswerter lesefeldAuswerter = new LesefeldAuswerter();
+        lesefeldAuswerter.setLesefelderDefinitionsFromDatabase();
+        lesefeldAuswerter.setMap(new MitgliedMap().getMap(new MitgliedDummy(), null));
```

Comment:
du kannst ein neues, leeres Mitgleid erzeugen und es der Mitgliedmap übergeben, dann werden automatisch die Dummyeinträge verwendet, da ist keine extra Klasse notwendig. Ggfs. kannst du die Dummydaten in Mitgliedmap erweitern.

Reply:
Das hat bei mir eine Exception geworfen (NotInitialized oder so), habe es daher ausgelagert. Hätte eher andersrum gesagt, dass ich in MitgliedMap statt dort alle Dummy-Werte zu setzen einfach nur ein DummyMitglied erzeuge. Meiner Meinung nach ist das die elegantere Weise, falls in Zukunft woanders nochmal Dummy Werte gebraucht werden, kann man die an einem Ort verwalten.